src/services/recoveryVaults/schema.graphql

```diff
@@ -45,12 +45,43 @@ type azureRecoveryVaultPrivateEndpointConnectionVaultProperties
   )
   @key(fields: "id") {
   id: String! @id @search(by: [hash])
-  properties: azureRecoveryVaultPrivateEndpointConnection
+  properties: azureRecoveryVaultPrivateEndpointConn
   name: String @search(by: [hash, regexp])
   type: String @search(by: [hash, regexp])
   location: String @search(by: [hash, regexp])
 }
 
+type azureRecoveryVaultPrivateEndpointConn
+  @generate(
+    query: { get: false, query: true, aggregate: false }
+    mutation: { add: false, delete: false }
+    subscription: false
+  ) {
+  provisioningState: String @search(by: [hash, regexp])
+  privateEndpoint: azureRecoveryVaultPrivateEndpoint
+  privateLinkServiceConnectionState: azureRecoveryVaultPrivateLinkServiceConnectionState
```

Comment:
is this name not too long here? `azureRecoveryVaultPrivateLinkServiceConnectionState`

Reply:
Not in this case since it's not part of a many-to-many relationship. That is where the name composed by the two tables, generally, exceeds the maximum limit allowed.